### ISSUE TITLE
Language is not supported when it contains a ‘-’ character

### ISSUE THREAD

**Issue by AhmedElwerdany** [AhmedElwerdany/blogger-image#2]
Sorry Ahmed for disturb you again.
I've made a fresh install on a server and this time I've got Chromium in english and I see Blogger menu in english.
Anyway I have got this error:
" throw new Error('Sorry, language is not supported\nlanguage: ' + language);
                  ^

Error: Sorry, language is not supported
language: en-GB
    at Uploader.init (file:///home/cirillo-server/Desktop/OPENROBO/blogger-image/node_modules/blogger-image/build/index.js:64:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///home/cirillo-server/Desktop/OPENROBO/blogger-image/index.js:18:1
"

Can I help you in some way?
Thanks alot

_Originally posted by @francescocassini in https://github.com/AhmedElwerdany/blogger-image/issues/1#issuecomment-1835690938_

**Comment by francescocassini** [comment]
It WORKS!!!
I must confess, though, that maybe yesterday it was my fault. In fact I had only done "npm uninstall blogger-image" and after that I reinstalled it with "npm i blogger-image".
Instead this morning I deleted the folder and re-downloaded itfrom github: after that I did "npm i blogger-image"... unfortunately I am not very familiar with node.js
From then on it worked perfectly!
Not only that: I created other Blogger blogs where I set the language to German and French and they work.
Furthermore, I think I also identified WHERE you have to set the language correctly in Chromium: when you first log into Chromium in google (and not in Blogger, but in GMAIL!). In fact in the google interface there is the language setting and I changed and put the Italian language: that is the language that needs to be set correctly, not inside the Blogger blog settings.
Thanks again for your incredible work: after I will make other test with video other than image!!!

**Comment by AhmedElwerdany** [comment]
Hi @francescocassini ,

I’m glad to hear that everything is working well for you with the image upload functionality. However, I wanted to let you know that the package may not support video uploads. This is because Blogger treats videos differently from images. Specifically, the URL for a video is only available after it has been rendered on a blog post. Therefore, attempting to upload videos using this package might not yield the expected results.

I hope this information is helpful. Please feel free to reach out if you have any other questions or concerns.

**Comment by francescocassini** [comment]
You are right Ahmed, in fact yesterday morning I had already done a test, but I had received an error with a little video.
However, it is not a problem because I can upload the videos to Youtube for free.
And your program remains a masterpiece and it is absolutely UNIQUE, because I have done so much research and I have not found any other like it.
You are really great!

**Comment by AhmedElwerdany** [comment]
@francescocassini 
Thank you for your encouraging words
